src/systests/test_userbehavior.py

```diff
@@ -100,3 +104,24 @@ def check_elasticsearch_response(self, response, es_query):
         print(response)
         timestamp = json.loads(response)["hits"]["hits"][0]["_source"][es_query.timestamp_key]
         return self.is_iso8601_timestamp(timestamp)
+
+    @pytest.mark.parametrize("client_id", client_ids, ids=lambda x: "Client {}".format(x))
+    def test_client_userbehavior_is_running(self, client_id, timeout_counter):
+        command = "powershell -c \"Get-WmiObject Win32_process\""
+        success_indicator = "CommandLine                : python C:\\BREACH\\userbehavior\\run.py\r\n"
```

Comment:
Can we be sure that the number of spaces in the `success_indicator` is always the same? Maybe change?

Reply:
Running the command in question (`powershell -c "Get-WmiObject Win32_process"`) directly on the machine seems to always return the same output in the same way:
```
[...]
Caption                    : python.exe
CommandLine                : python C:\BREACH\userbehavior\run.py --use-breach-setup
CreationClassName          : Win32_ComputerSystem
CSName                     : CLIENT1
[...]
```
Notably, the spacing until the colon seems to be consistent for *every* kind of output `Get-WmiObject` produces. We could argue that this might - possibly - change in the future, and since the latter part of our search string only appears in the `CommandLine` field anyway (during my tests that is), we could drop its name including the spaces and just use
```
success_indicator =  "python C:\\BREACH\\userbehavior\\run.py\r\n"
```
or, to also include the provided argument and skip the escape sequences because why not
```
success_indicator =  "python C:\\BREACH\\userbehavior\\run.py --use-breach-setup"
```
The `\r\n` didn't cause any trouble before because Sebastian has adapted our functions that check for certain output to ignore ANSI escape sequences (`try_until_counter_reached` and `check_printed`), which is nice.

The slightly more complicated solution would be to use a regexp within our assertion function to catch any deviations, but eh `¯\_(ツ)_/¯`

Let me know what the best solution is, then I'll push it, run another pipeline and notify you when it succeeds.